terraform/gcp/main.tf

```diff
@@ -30,7 +30,7 @@ resource "google_compute_firewall" "node-exporter-{{ name }}" {
 
   allow {
     protocol = "tcp"
-    ports    = ["9100"]
+    ports    = ["9100","9616"]
```

Comment:
Same here:

> Does this only get enabled when `node_exporter_enabled` is set to `true`?

Reply:
Actually, isn't the point that the `9100` port is hidden behind the reverse proxy (`9616`)? So why allow both?

Reply:
Nevermind: https://substrate.dev/docs/en/tutorials/visualize-node-metrics/#start-a-substrate-node

Two different ports for two different purposes :)